src/docker-compose.yaml

```diff
@@ -39,7 +39,7 @@ services:
             - ./environment/dev/app/supervisord.conf:/etc/supervisor/custom-supervisord.conf:ro
             - .:/application
         ports:
-            - ${DOCKER_APP_HOST_PORT:-3851}:80
+            - ${DOCKER_APP_HOST_PORT:-63851}:80
```

Comment:
What is the reason?

Reply:
because default ports should be the same as in `.env.example` in my opinion

Reply:
Okay, but these should be also rewritten into ports provided by user. Please update `init.sh`.